solr/core/src/test-files/solr/collection1/conf/solrconfig-nodelevelcaches.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" ?>
+
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one or more
+ contributor license agreements.  See the NOTICE file distributed with
+ this work for additional information regarding copyright ownership.
+ The ASF licenses this file to You under the Apache License, Version 2.0
+ (the "License"); you may not use this file except in compliance with
+ the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<config>
+  <luceneMatchVersion>${tests.luceneMatchVersion:LATEST}</luceneMatchVersion>
+  <dataDir>${solr.data.dir:}</dataDir>
+  <xi:include href="solrconfig.snippet.randomindexconfig.xml" xmlns:xi="http://www.w3.org/2001/XInclude"/>
+  <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.RAMDirectoryFactory}"/>
+  <schemaFactory class="ClassicIndexSchemaFactory"/>
+  <requestHandler name="/select" class="solr.SearchHandler" />
+  <query>
+    <filterCache
+      class="solr.ThinCache"
+      parentCacheName="myNodeLevelCacheThin"
```

Comment:
myNodeLevelCacheThin needs to define in solr.xml?

Reply:
yes, exactly.

Reply:
What is the motivation to keep this separate (not in solrconfig)? Would be good to have `cacheref` element in `colrconfig` ?

Reply:
>What is the motivation to keep this separate (not in solrconfig)?

The cache is configured in the config file that's associated with the same scope as the cache itself. `solrconfig.xml` configures properties associated with the collection/core. A node-level cache is configured in the node-level config file, `solr.xml`.

> What is the motivation to keep this separate (not in solrconfig)? Would be good to have cacheref element in colrconfig ?

I'm not sure I understand?

Reply:
I feel like both should be configure in one place ? probably its easier to track